### PR TITLE
Integrate the litmus images with scarf gateway

### PR DESCRIPTION
**Pull request by ispeakc0de**
#### What this PR does / why we need it:

Litmus is making use of [Scarf](https://docs.scarf.sh/) to obtain usage stats (via the docker images) to measure the adoption of the project. This requires adding the scarf gateway prefix to the images in the deployment specs. This PR instruments the Litmus Helm charts with the same. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Review by Saranya-jena — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Review by neelanjan00 — APPROVED** [review]

**Comment by chandankumar4** [comment]
Is that deploy chart failure is expected behaviour?

**Comment by ksatchit** [comment]
> Is that deploy chart failure is expected behaviour?

Shouldn't be/ seems like a CRD install issue that needs to be looked into

**Review by Jasstkn — APPROVED** [review]
LGTM

File: ct.yaml [hunk 1]
```diff
@@ -2,3 +2,10 @@ chart-dirs:
   - charts
 chart-repos:
   - bitnami=https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
+charts:
```
Comment (Jasstkn) [inline ct.yaml:5]: charts excluded in the github workflow

Comment (Jasstkn) [reply ct.yaml:5]: it's here: https://github.com/litmuschaos/litmus-helm/blob/master/.github/workflows/test.yml#L75

**Pull request merged**